wordpress-mobile/WordPress-iOS
# Feature/7616 drag n drop

Fixes #7616 
This is a partial implementation of 7616. Included in this PR:
- Dragging a photo onto the site icon in the header of BlogDetailViewController to set a new site icon.
This means dragging a single image from an outside source and dropping it onto the site icon. 

- Dragging a photo onto the user's gravatar (in the Me section) to update it.
This means dragging a single image from an outside source and dropping it onto the gravatar.

To test:
(These steps can be used for either the gravatar or the site icon)
1. Open WordPress-iOS on an iPad or an iPad simulator of your choice.
2. Open Photos in a multi-task window. (The two apps should be "snapped" side by side).
3. Multi-touch to grab multiple photos. 
- Drag them over to the gravatar and attempt to drop them.
- Observe that the gravatar does nothing, the action is canceled.
4. Long-press a single photo from the Photos app and drag it over the gravatar (in the Me section). 
5. Observe that the gravatar has an animation that reacts to the photo being dragged over the active area. // Note: there is no animation on the site icon.
6. Drop the photo onto the gravatar.
7. A "Crop and Use" picker dialog should appear.
- Choose Cancel > dismisses the picker and does not change the gravatar.
- Choose Use > crops the photo and uploads the image.
